builder/upload.sh

```diff
@@ -53,6 +60,16 @@ export HOME="$PWD"
 jq -r '.registry_password' "$SECRETS" | \
         oras login -u "$(jq -r '.registry_user' "$SECRETS")"  \
         --password-stdin "$REGISTRY"
-oras push --artifact-type application/application-x-qemu-disk \
-       --annotation-file "$OUTPUT_DIR/annotation.json" \
-        "$REGISTRY/$PROJECT/$REPOSITORY:$TAG" "$QCOW_FILE"
+
+echo "## debugging"
+echo "REPOSITORY: $REPOSITORY"
+echo "VM_NAME: $VM_NAME"
+echo "TAG: $TAG"
+echo "QEMU_SOURCE_ID: $QEMU_SOURCE_ID"
+echo "IMAGE: $IMAGE"
+echo "QCOW_FILE: $QCOW_FILE"
+ls -l "$QCOW_FILE"
+ls -l "metadata.json"
```

Comment:
Do we want to leave this?

Reply:
I was thinking to leave just in case it becomes needed, but we can remove it